course/theme.json

```diff
@@ -670,6 +670,9 @@
 				},
 				"css": "& .sensei-lms-course-navigation-module__summary { font-family: var(--wp--preset--font-family--system); }"
 			},
+		"sensei-lms/course-theme-notices": {
+				"css": "& .sensei-course-theme-quiz-graded-notice__title { font-family: var(--wp--preset--font-family--system); font-size: clamp(1.5rem, 1.167rem + 1.111vw, 2rem); font-weight: 590; } .sensei-course-theme-quiz-graded-notice__text { font-size: clamp(1.25rem, 1.083rem + 0.556vw, 1.5rem); line-height: 1.3; } .sensei-course-theme-quiz-graded-notice__reset-quiz-form .sensei-course-theme__button { font-size: clamp(1.313rem, 1.188rem + 0.417vw, 1.5rem); }"
+			},
```

Comment:
Do you think converting this title to a heading element like `h3` can be a good idea rather than styling the div content to make it look like a title heading? Then we can also style it without CSS by using the elements->h3 property here. That'll also probably make `.sensei-course-theme-quiz-graded-notice__text` stylable using the 'typography' property instead.

Reply:
Nice suggestion! Changed https://github.com/Automattic/themes/pull/7401/commits/8e3c4e4f9727b2ec08c7bbd97c00f5f96634f8ca, https://github.com/Automattic/sensei/pull/7197/commits/57f6d8a789ed3c916dd2dcc101c38b5fd40443f5 and [here](https://github.com/Automattic/sensei/pull/7197/commits/d3ddc7b24d9b0ceb8b19b50aedb1ddc39a7865fb).

Note that I made it an `h2` in order for the HTML to be semantic.